mycore-viewer/src/main/typescript/modules/mobile/widgets/toolbar/view/button/MobileButtonView.ts

```diff
@@ -47,14 +47,14 @@ namespace mycore.viewer.widgets.toolbar {
         }
 
         public updateButtonIcon(icon:string):void {
-            this._buttonIcon.removeClass("fa");
-            this._buttonIcon.removeClass("fa-" + this._lastIcon);
-            this._buttonIcon.removeClass("icon-" + this._lastIcon);
+            this._buttonIcon.removeClass(`fas`);
```

Comment:
schräges, statt gerades Anführungszeichen in diesem Block?

Reply:
Ja das ist in dem Fall egal. Mit  ` macht man Template-Strings und mit ' normale.